### DISCUSSION TITLE
Specific Task Help/Formatting Help (big noob here)

### DISCUSSION THREAD

**Discussion by SpookyMcBones** [tfeldmann/organize#290] [Q&A]
Hello, you will all have to forgive me, as I am a huge noob to not just this program but coding all together.

I would like to delete all files in folders with a certain name.

Despite reading lots in the documentation, I also struggle to understand the formatting of the scripting language - e.g. how certain lines need a number of spaces, or maybe a dash, to function, otherwise the script breaks. I thought these were just for readability, but I was proven wrong.

**Comment by tfeldmann** [comment]
Can you describe what you want to achieve? Maybe with a small example?
